apps/website/screens/components/flex/code/FlexCodePage.tsx

```diff
@@ -130,14 +128,14 @@ const sections = [
         <tr>
           <td>gap</td>
           <td>
-            <TableCode>{coreSpacingTokensTypeString} | Gap</TableCode>
+            <TableCode>string | Gap</TableCode>
```

Comment:
We can specify the `Gap` type in the type column instead of the description, as we do in other components (look at the Breadcrumbs):

![image](https://github.com/user-attachments/assets/cc68b5f5-a105-4b4e-92e2-bdb643d011a5)

I think it is cleaner.